liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -964,6 +964,20 @@ private void installEmptyEAR(MavenProject earProject) throws MojoExecutionExcept
         }
     }
 
+    protected void purgeLocalRepositoryModule(MavenProject project) throws MojoExecutionException {
+        Plugin plugin = getPlugin("org.apache.maven.plugins", "maven-dependency-plugin");
```

Comment:
Consider using `getPluginForProject` to ensure that if the plugin is specified in the `pluginManagement` block of pom.xml, it is referencing the pom of the specified MavenProject (not the main project liberty:dev is running on)

```java
Plugin plugin = getPluginForProject("org.apache.maven.plugins", "maven-dependency-plugin", project);
```

Reply:
Thanks for pointing that out.  The plugin really only needs to run for the current project (when `dev` is invoked on the upstream module during the initial Reactor build).  So I removed the `MavenProject` parameter from this method instead.